Tasks/AzureRmWebAppDeployment/AzureRmWebAppDeployment.ps1

```diff
@@ -100,6 +115,13 @@ try{
 	Write-Verbose "Completed AzureRM WebApp Deployment Task"
 
 } finally {
+    if( $VariableSubstitution -eq "true")
+    {
+        if( Test-Path $packageFilePath )
+        {
+            Remove-Item -Force $packageFilePath
```

Comment:
I believe you wanted to use rmdir as Remove-Item was causing path limit exceeding error.